chart/tests/test_pod_template_file.py

```diff
@@ -135,6 +135,41 @@ def test_validate_if_ssh_params_are_added(self):
             jmespath.search("spec.volumes", docs[0]),
         )
 
+    def test_validate_if_ssh_known_hosts_are_added(self):
+        docs = render_chart(
+            values={
+                "dags": {
+                    "gitSync": {
+                        "enabled": True,
+                        "containerName": "git-sync-test",
+                        "sshKeySecret": "ssh-secret",
+                        "knownHosts": "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==",
```

Comment:
One of the static check is failing: https://github.com/apache/airflow/pull/13457/checks?check_run_id=1662277696#step:7:200

This should fix it

```suggestion
                        "knownHosts": "github.com ssh-rsa AAAABdummy",
```